File: app/src/main/java/com/matf/filemanager/UtilClasses/SaveStatus.java
package com.matf.filemanager.UtilClasses;

public enum SaveStatus {
    FILESAVED,
    ERRORSAVING,
    FILENOTCHANGED;

    @Override
    public String toString() {
        if(this == SaveStatus.FILESAVED){
            return "File saved!";
        }else if (this == SaveStatus.ERRORSAVING){
            return "An error has occurred!";
        }else{
            return "There are no changes to save!";
        }
    }
}

File: app/src/main/java/com/matf/filemanager/UtilClasses/SaveStatus.kt
package com.matf.filemanager.UtilClasses

enum class SaveStatus {
    FILESAVED, ERRORSAVING, FILENOTCHANGED;

    override fun toString(): String {
        return when {
            this == FILESAVED -> {
                "File saved!"
            }
            this == ERRORSAVING -> {
                "An error has occurred!"
            }
            else -> {
                "There are no changes to save!"
            }
        }
    }
}
